apps/src/weblab/brambleHost.js

```diff
@@ -65,6 +68,100 @@ function putFilesInBramble(Bramble, sources, callback) {
   });
 }
 
+function removeAllAssetsInBramble(Bramble, callback) {
```

Comment:
Yes, this was a case where I copied the same signature and then later realized that it was not needed. I'll clean that up.